src/backend/gl/src/command.rs

```diff
@@ -147,8 +147,8 @@ pub struct RenderPassCache {
 struct Cache {
     // Active primitive topology, set by the current pipeline.
     primitive: Option<u32>,
-    // Active index type, set by the current index buffer.
-    index_type: Option<hal::IndexType>,
+    // Active index type and buffer range, set by the current index buffer.
+    index_type_range: Option<(hal::IndexType, Range<u64>)>,
```

Comment:
we could use `hal::buffer::Offset` typedef here

Reply:
good catch, done!